.travis.yml

```diff
@@ -7,11 +7,10 @@ script: bundle exec rake travis
 matrix:
   fast_finish: true
   include:
-    - rvm: 1.9.3
```

Comment:
Are you sure? https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/